src/Compilation.zig

```diff
@@ -1734,7 +1733,9 @@ pub fn create(gpa: Allocator, options: InitOptions) !*Compilation {
                 try comp.work_queue.writeItem(.{ .libssp = {} });
             }
 
-            if (!comp.bin_file.options.link_libc and capable_of_building_zig_libc) {
+            if (!comp.bin_file.options.link_libc and capable_of_building_zig_libc and
```

Comment:
Could this be the culprit of the CI failures?

Reply:
Ah yes, i misinterpreted the difference between libc and zig_libc.

Reply:
Yea that was it